artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py

```diff
@@ -505,6 +505,12 @@
     "Directory Traversal vulnerability in FileMage Gateway Windows Deployments v.1.10.8 and before allows a remote attacker to obtain sensitive information via a crafted request to the /mgmt/ component.": "FileMage Gateway w wersji v.1.10.8 i wcześniejszych zawiera podatność Directory Traversal umożliwiającą atakującemu pobieranie dowolnych plików z serwera.",
     "http/misconfiguration/installer/prestashop-installer.yaml": "Wykryto instalator systemu PrestaShop. Rekomendujemy, aby takie zasoby nie były dostępne publicznie, ponieważ mogą umożliwić atakującemu uzyskanie nieuprawnionego dostępu do systemu czy zmiany w jego konfiguracji.",
     "http/takeovers/vercel-takeover.yaml": "Wykryto domenę skonfigurowaną, aby serwować treści z serwisu Vercel, ale konto docelowe nie istnieje. Jeśli domena nie jest używana, rekomendujemy jej usunięcie, aby atakujący nie mógł zarejestrować domeny w serwisie Vercel i serwować swoich treści.",
+    "Microweber before 1.1.20 is susceptible to information disclosure via userfiles/modules/users/controller/controller.php. An attacker can disclose the users database via a /modules/ POST request and thus potentially access sensitive information, modify data, and/or execute unauthorized operations.": "Narzędzie Microweber w wersji poniżej 1.1.20 umożliwia atakującemu pobranie danych użytkowników, a w konsekwencji dostęp do informacji wrażliwych i potencjalnie zmianę danych i wykonanie nieuprawnionych operacji.",
+    "Amcrest IPM-721S V2.420.AC00.16.R.20160909 devices allow an unauthenticated attacker to download the administrative credentials.": "Urządzenia Amcrest IPM-721S V2.420.AC00.16.R.20160909 umożliwiają nieuwierzytelnionemu atakującemu pobranie danych dostępowych umożliwiających logowanie na konto administratora.",
+    "Mida eFramework contains a cross-site scripting vulnerability. An attacker can execute arbitrary script in the browser of an unsuspecting user in the context of the affected site. This can allow the attacker to steal cookie-based authentication credentials and launch other attacks.": "Wykryto podatność Cross-Site Scripting w narzędziu Mida eFramework. Ta podatność umożliwia atakującemu spreparowanie linku, który, gdy zostanie kliknięty przez administratora, umożliwi atakującemu zdobycie uprawnień administratora.",
```

Comment:
```suggestion
    "Mida eFramework contains a cross-site scripting vulnerability. An attacker can execute arbitrary script in the browser of an unsuspecting user in the context of the affected site. This can allow the attacker to steal cookie-based authentication credentials and launch other attacks.": "Wykryto podatność Cross-Site Scripting w narzędziu Mida eFramework. Ta podatność umożliwia atakującemu spreparowanie linku, który, gdy zostanie kliknięty przez administratora, umożliwi wykonanie akcji z jego uprawnieniami.",
```